build.gradle

```diff
@@ -72,7 +72,7 @@ subprojects { project ->
     apply plugin: "groovy"
     if (project.name.startsWith("examples")) {
         apply plugin: "org.grails.grails-web"
-        apply plugin: "org.grails.plugins.views-json"
+        //apply plugin: "org.grails.plugins.views-json"
```

Comment:
Not required for tests here, issue:  https://github.com/grails/grails-views/issues/587

Reply:
I think it is used to compile `examples/demo33/grails-app/views/json/index.gson`